web/styles/settings.css

```diff
@@ -2065,6 +2065,10 @@ $option_title_width: 180px;
             margin-right: 12px;
             margin-top: 12px;
         }
+
+        & .dropdown_widget_with_label_wrapper {
+            height: 30px;
+        }
```

Comment:
I adjusted height of the wrapper wherever the `dropdown-widget-button` height is changed.